server/server.go

```diff
@@ -217,40 +217,28 @@ func NewServer(config *Config) (*Server, error) {
 	}
 
 	// apply allow list contracts deployer genesis data
-	if m.config.Chain.Params.ContractDeployerAllowList != nil {
-		addresslist.ApplyGenesisAllocs(m.config.Chain.Genesis, contracts.AllowListContractsAddr,
-			m.config.Chain.Params.ContractDeployerAllowList)
-	}
+	addresslist.ApplyGenesisAllocs(m.config.Chain.Genesis, contracts.AllowListContractsAddr,
```

Comment:
Enabling the ACL at genesis time unconditionally is not a good idea, it implies extra gas cots for all users regardless if they want to use the ACLs or not

Reply:
First line in function is
```
	if superAdmin == nil && config == nil {
		return
	}
	```

Reply:
I prefer to have checks on one place instead on six, but i can change that if needed